privacy_evaluator/metrics/privacy_risk_score.py

```diff
@@ -0,0 +1,72 @@
+from privacy_evaluator.classifiers.classifier import Classifier
+import numpy as np
+
+
+def compute_privacy_risk_score(
```

Comment:
Same as below: Can you please add the return types here (e.g. `compute_privacy_risk_score(...) -> Tupel[np.ndarray, np.ndarray]:`)